fixed bug where aggregating ignored --keep_decoys flag

jspaezp opened the pull request:
welp ... I missed this when I submitted https://github.com/wfondrie/mokapot/pull/32

but there was a bug where the --keep_decoys argument was ignored when aggregating the output.
sorry about that ...

wfondrie commented:
Thanks - I should have caught this in my review! The patch looks good to me, but I think we should add an additional test case to `test_cli_aggregate()`: https://github.com/wfondrie/mokapot/blob/0c099860e6f089b1754015ce0e9c07b4a070af05/tests/system_tests/test_cli.py#L99-L116

Can you add the `--keep-decoys` flag to the command and test that the decoy files exist, similar to what I added to in `test_cli_options()`?

jspaezp commented:
Added in 0542a5fbe589abde41a4fb70fc2d1d70037ce012 :) Also added a test so the decoy output is not in the output when not asked for (lack of --keep_decoys flag)

wfondrie commented:
This looks great! I'll merge it and draft a new release.
